idaes/power_generation/carbon_capture/mea_solvent_system/unit_models/column.py

```diff
@@ -0,0 +1,1712 @@
+                             doc="Packing void space m3/m3")
+
+        self.a_ref = Param(initialize=self.config.packing_specific_area,
+                           units=pyunits.m**2 / pyunits.m**3,
+                           doc="Packing specific surface area m2/m3")
+
+        self.dh_ref = Expression(expr= 4 * self.eps_ref /self.a_ref,
+                                 doc="Hydraulic diameter")
+
+        # specific constants for volumetric mass transfer coefficients
+        # reference:  Billet and Schultes, 1999
+        self.Cv_ref = Var(initialize=0.357,
+                          doc='''Vapor packing specific constant in
+                                Billet and Schultes' (1999) volumetric
+                                mass transfer coefficient correlation''')
+        self.Cl_ref = Var(initialize=0.5,
+                          doc='''Liquid packing specific constant in
+                                Billet and Schultes' (1999) volumetric
+                                mass transfer coefficient correlation''')
+        self.Cv_ref.fix()
+        self.Cl_ref.fix()
+
+        # Add object references - others
+        R_ref = CONST.gas_constant
+
+        # Unit Model Parameters/sets
+        self.zi = Param(self.vapor_phase.length_domain, mutable=True,
+                        doc='''Integer indexing parameter required for transfer
+                             across boundaries of a given volume element''')
+        # Set the integer  indices
+        for i, x in enumerate(self.vapor_phase.length_domain, 1):
+            self.zi[x] = i
+
+        # Continuation parameters for initialization
+        self._homotopy_par_m = Param(initialize=0, mutable=True, units=None,
+                                     doc='''Continuation parameter to turn on mass
+                                     transfer terms gradually''')
+        self._homotopy_par_h = Param(initialize=0, mutable=True, units=None,
+                                     doc='''Continuation parameter to turn on heat
+                                     transfer terms gradually''')
+
+        # fixed column pressure
+        if self.config.fix_column_pressure:
+            self.column_pressure = Param(initialize=self.config.column_pressure,
+                                         mutable=True,
+                                         units=pyunits.Pa,
+                                         doc='Fixed operating pressure of column')
+
+        # Interfacial area  parameters
+        self.area_interfacial_parA = Var(initialize=0.6486,
+                                         units=None,
+                                         doc='''Interfacial area parameter A''')
+
+        self.area_interfacial_parB = Var(initialize=0.12,
+                                         units=None,
+                                         doc='''Interfacial area parameter B''')
+        self.area_interfacial_parA.fix(0.6486)
+        self.area_interfacial_parB.fix(0.12)
```

Comment:
Same comment as previous regarding fixing vars in the model.

Reply:
@jghouse88, same as above, these might be used for fitting the correlation for the interfacial area.